website/docs/backends/types/remote.html.md

```diff
@@ -67,7 +75,8 @@ terraform {
     organization = "company"
 
     workspaces {
-      name = "workspace"
+      name = "my-app-prod"
+      # OR:
```

Comment:
I think it might be clearer to keep these fully separate, at the expense of slight verbosity. Just a thought.